testcases/BootTorture.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2015,2017
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+# Torture the machine with repeatedly trying to boot
+
+import unittest
+
+import OpTestConfiguration
+from common.OpTestUtil import OpTestUtil
+from common.OpTestSystem import OpSystemState
+
+class BootTorture(unittest.TestCase):
+    def setUp(self):
+        conf = OpTestConfiguration.conf
+        self.system = conf.system()
+
+    def runTest(self):
+        for i in range(1,1024):
+            print "Boot iteration %d..." % i
+            self.system.goto_state(OpSystemState.OFF)
+            self.system.goto_state(OpSystemState.PETITBOOT)
+
```

Comment:
Here for each iteration, once system reaches to petitboot, getting the list of errors in OPAL and Skiroot kernel will be good. We can grep for opal errors as it is in OpalMsglog test. But i am not sure how to grep kernel errors.